cachito/workers/cleanup_job.py

```diff
@@ -21,6 +21,7 @@
 
 
 def main():
+    logging.basicConfig(level=logging.INFO)
     """Mark all end of life requests as stale using the REST API."""
```

Comment:
Let's move the basicConfig call below the docstring

Reply:
Got it, I initially put it above the docstring because I thought it directly pertained to the lines directly below.

Reply:
Right, docstrings should always be the first thing under the name of the function (unlike comments, which can be anywhere)